runtime/compiler/codegen/J9RecognizedMethodsEnum.hpp

```diff
@@ -1006,7 +1006,7 @@
    java_lang_invoke_MethodHandle_invoke,
    java_lang_invoke_MethodHandle_invokeExact,
    java_lang_invoke_MethodHandle_invokeExactTargetAddress,
-   java_lang_invoke_MethodHandle_invokeWithArgumentsHelper,
+   java_lang_invoke_MethodHandleHelper_invokeWithArgumentsHelper,
```

Comment:
Rather than just renaming the enum entry - it would be better to add the new entry and alias it to the old one so we don't need to do a coordinated merge with the uses in OMR. Once we fix OMR you can do another change to remove the old unused enum entry. This is because there are a few bits of code that are Java specific in OMR that have not been refactored down to OpenJ9.